### PR TITLE
Practitioner portraits

### PR DESCRIPTION
**Pull request by lionsdwarf**

### CODE REVIEW

**Comment by lionsdwarf** [comment]
@craigspaeth I kept the string parsing really simple here. The practitioner's portrait filename can be included using backtics. 
screen shot 2018-10-01 at 6 48 01 pm

File: apps/portal/components/schedule-session-choose/schedule-session-choose.css [hunk 1]
```diff
@@ -58,3 +58,8 @@
   @apply --esteban-subheader;
   margin-bottom: calc(var(--incr) * 2);
 }
+
+.card img {
+  border-radius: 50%;
```
Comment (craigspaeth) [inline apps/portal/components/schedule-session-choose/schedule-session-choose.css]: Onice, I didn't think of using percents to achieve this before.

File: apps/portal/lib/practitioner.js [hunk 2]
```diff
@@ -44,36 +44,45 @@ export const cards = async () => {
...
+  let description = str.substring(0, str.indexOf('`'))
+    ? str.substring(0, str.indexOf('`')).trim()
+    : str
+  let portraitFilename = str.substring(0, str.indexOf('`'))
+    ? str.substring(str.indexOf('`') + 1, str.length - 1).trim()
+    : ''
+  return { description, portraitFilename }
+}
+
+export const buildCards = arr =>
```
Comment (craigspaeth) [inline apps/portal/lib/practitioner.js]: Nice use of pure functions to handle the data manipulation here!

File: apps/portal/lib/test/practitioner.test.js [hunk 3]
```diff
@@ -1,29 +1,53 @@
...
+test(`'parseDescriptionAndPortraitFilename' returns a description and a portrait filename when a portrait filename is present in backticks`, () => {
+  const {
+    description,
+    portraitFilename
+  } = Practitioner.parseDescriptionAndPortraitFilename(practitionerDescription)
+  expect(description).toEqual('Some meaningless verbiage to parse.')
+  expect(portraitFilename).toEqual('withFilename.jpg')
+})
+
+test(`'parseDescriptionAndPortraitFilename' returns a description and a null portrait filename when a portrait filename is not present`, () => {
```
Comment (craigspaeth) [inline apps/portal/lib/test/practitioner.test.js]: ❤️ tests!

**Comment by craigspaeth** [comment]
Good stuff! Really like how you broke out logic into lib/practitioner.

**Pull request merged**